feat: add dedicated top-level pages per challenge

alexander-held opened the pull request:
Resolves #2421 

Create new top-level challenge pages to (in particular) better group AGC / IDAP projects.

alexander-held committed:
add dedicated top-level pages per challenge

alexander-held committed:
add links to subpages

alexander-held committed:
remove old text

oshadura committed:
Add challenge-area layout for grand challenges

oshadura committed:
Swicth other challenges to use layout and fix navigation bar

oshadura committed:
Add 200 gbps project

oshadura committed:
Add short name for agc

oshadura committed:
Fix typos

oshadura committed:
Fix the links

oshadura committed:
Add people to challenge area

oshadura committed:
Add people to challenge area

oshadura committed:
Update 200 gbps page

oshadura committed:
Challenge area should match the name of md file

oshadura committed:
Try to test if build is stuck

oshadura committed:
Add agc project under challenge

oshadura committed:
Add challenge_area to people.schema.json

rct225 committed:
Update people.schema.json

Adding challenge-area to object properties

oshadura commented:
<img width="1188" alt="Screenshot 2024-12-10 at 15 25 13" src="https://github.com/user-attachments/assets/12da5176-1a6b-4919-8221-5c5c6304f9de">

@rct225 I tried to rebuild and it looks the same for me :-(

rct225 committed:
Merge branch 'master' into refactor/challenges

rct225 commented:
Are we ready to push this out?

matthewfeickert commented:
> Are we ready to push this out?

I would assume no, given @oshadura's comment https://github.com/iris-hep/iris-hep.github.io/pull/2437#issuecomment-2531788935.

rct225 commented:
Ah … misread it.

oshadura committed:
Rename area name as agc and fix showing AGC projects and AGC team members

oshadura commented:
<img width="834" alt="Screenshot 2025-01-06 at 15 22 17" src="https://github.com/user-attachments/assets/78e6b7c2-1fc3-4f6f-b918-d32f149730c3" />

oshadura committed:
Remove videos which are not challenge related

oshadura committed:
Remove gannt chart for challenge

oshadura committed:
Tring to fix publication lists

pre-commit-ci[bot] committed:
style: pre-commit fixes

rct225 committed:
Update checkpubs.rb

Does disabling the LineLength cop fix this issue?

rct225 committed:
Update checks_extend.rb

adding function for challenge_areas

rct225 committed:
Update checks_extend.rb

rct225 committed:
Update checkpubs.rb

rct225 committed:
Update checks_extend.rb

rct225 committed:
Update data-grand-challenge.md

rct225 committed:
Update training-grand-challenge.md

rct225 committed:
Update getpub.rb

Forgive me is this is a dumb question.  Do we want to force every publication to have a challenge-area?  What about puts that existed before the challenges?

pre-commit-ci[bot] committed:
style: pre-commit fixes

rct225 committed:
Update getpub.rb

pre-commit-ci[bot] committed:
style: pre-commit fixes

rct225 committed:
Update getpub.rb

rct225 commented:
@alexander-held @oshadura I was trying a few things to get the page to build.  One question I have was do we want to force all publications to have a challenge-area?  I commented the block out requiring it in getpub.rb since most of the YML files for publication do not have an entry for challenge-area.

oshadura commented:
Hi @rct225! Thank a lot, now it looks like finally works as we wanted! There is tiny rendering issue, but otherwise perfect!
<img width="1014" alt="Screenshot 2025-01-15 at 14 23 34" src="https://github.com/user-attachments/assets/2eadd1cd-e2fc-4d55-a19d-165d2b7839b6" />

rct225 commented:
Happy to help!

oshadura committed:
Add challenge area to presentations

oshadura commented:
@rct225 could I ask you please to take a look (if you will have time) how to add presentations as well to list?
That will fix rendering issue and then after we should be ready to merge it 🥇

rct225 committed:
Update people.schema.json

rct225 committed:
Update people.schema.json

Typo - "challenge_area" instead of "challenge_areas" in presentation definition block

rct225 commented:
@oshadura @alexander-held I think I fixed the problem with my latest update.  Seems to work.

rct225 committed:
Update people.schema.json

Added "training-grand-challenge" just for completeness

oshadura committed:
Improve rendering of page

oshadura commented:
@rct225 @alexander-held now it looks perfect! 
(I just improved a bit rendering of web-page, some divs were overlapping for me and I could see some content).

<img width="1017" alt="Screenshot 2025-01-17 at 10 58 03" src="https://github.com/user-attachments/assets/922cd566-d483-4907-98ec-4478c95610c9" />
<img width="1163" alt="Screenshot 2025-01-17 at 10 57 52" src="https://github.com/user-attachments/assets/f26502b6-8f56-4ce3-8f82-df370260d8d1" />

oshadura commented:
@rct225 I think we can merge it and we will finish updating presentations/publications in the next iteration.

oshadura committed:
Merge branch 'master' into refactor/challenges

alexander-held reviewed (commented):
Great, thank you very much! This looks all good from my side.
